manifests/resolver.pp

```diff
@@ -87,11 +96,13 @@
 # Ricardo Rosales <missingcharacter@gmail.com>
 #
 define haproxy::resolver (
-  $nameservers             = undef,
+  Hash $nameservers       = {},
+  Boolean $parse_resolv_conf = false,
   $hold                    = undef,
   $resolve_retries         = undef,
   $timeout                 = undef,
-  $accepted_payload_size   = undef,
+  # https://cbonte.github.io/haproxy-dconv/1.8/configuration.html#5.3.2-accepted_payload_size
+  Optional[Integer[512, 8192]] $accepted_payload_size = undef,
```

Comment:
Could you align the different variable lengths

Reply:
Sure, done; I did this as an additional commit rather than mix functional and whitespace changes together.